math/tetration_mod/info.toml

```
@@ -6,3 +6,12 @@ timelimit = 10.0
 [[tests]]
     name = "random.cpp"
     number = 5
+[[tests]]
+    name = "max.cpp"
+    number = 5
```

```suggestion
    number = 3
```
ケースが弱いことがわかったらその時増やす感じにしたいです